dmrpp_generator/main.py

```diff
@@ -98,9 +98,17 @@ def get_bucket(filename, files, buckets):
                 break
         return buckets[bucket_type]
 
+    def _get_s3_extra(self):
+        """Helper to build the extra dict for S3 operations."""
+        extra = {}
+        if self.config.get("requester_pay", False):
```

Comment:
Do we need to parse the value as a boolean here?
```suggestion
        if f'{self.config.get("requester_pay", False)}.lower() == 'true':
```
or is the value in `self.config` already a boolean?